internal/services/desktopvirtualization/virtual_desktop_host_pool_resource_test.go

```diff
@@ -59,7 +68,7 @@ func TestAccVirtualDesktopHostPool_update(t *testing.T) {
 			),
 		},
 		{
-			Config: r.complete(data),
+			Config: r.complete(data, expirationTime),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
 				check.That(data.ResourceName).Key("tags.%").HasValue("1"),
```

Comment:
since we're validating that this has a value we should probably double-check that here?

```suggestion
				check.That(data.ResourceName).Key("tags.%").HasValue("1"),
				check.That(data.ResourceName).Key("registration_info.0.token").Exists(),
```

Reply:
Added to PR